### PR TITLE
Update test_dump-pcm_py.bats

### PR DESCRIPTION
**Pull request by zh794390558**
fix dump-pcm.py

### CODE REVIEW

**Comment by sw005320** [comment]
This causes the following error https://travis-ci.org/espnet/espnet/jobs/498039892#L1410-L1410
Could you explain your intention?

**Comment by zh794390558** [comment]
```
np.testing.assert_array_equal(test, test)
```

`test` equal `test`?

**Comment by sw005320** [comment]
I see. @kamo-naoyuki, could you follow up this?

File: test_utils/test_dump-pcm_py.bats [hunk 1]
```diff
@@ -38,7 +38,7 @@ with h5py.File('${tmpdir}/feats.h5') as h, open('${tmpdir}/wav.scp', 'r') as s:
         key, path = line.strip().split()
         valid, rate = W.read(path)
         test = h[key]
-        np.testing.assert_array_equal(test, test)
+        np.testing.assert_array_equal((rate, valid), test)
```
Comment (kamo-naoyuki) [inline test_utils/test_dump-pcm_py.bats]: Thanks.
```
assert rate == 8000
np.testing.assert_array_equal(test, valid)
```

Comment (zh794390558) [reply test_utils/test_dump-pcm_py.bats]: still not work.

File: test_utils/test_dump-pcm_py.bats [hunk 2]
```diff
@@ -38,7 +38,8 @@ with h5py.File('${tmpdir}/feats.h5') as h, open('${tmpdir}/wav.scp', 'r') as s:
         key, path = line.strip().split()
         valid, rate = W.read(path)
```
Comment (kamo-naoyuki) [inline test_utils/test_dump-pcm_py.bats]: Sorry, this is opposite.

```python
rate, valid = w.read(path)
```

**Comment by zh794390558** [comment]
Test Ok.

**Comment by zh794390558** [comment]
@sw005320

**Comment by sw005320** [comment]
thanks!

**Pull request merged**